activerecord/test/schema/schema.rb

```diff
@@ -1373,6 +1373,19 @@
     t.bigint :toooooooo_long_a_id, null: false
     t.bigint :toooooooo_long_b_id, null: false
   end
+
+  create_table :questions, force: :true do |t|
+    t.string :title
+    t.timestamps null: true
+  end
```

Comment:
```suggestion
  end

```

Knit: should probably have new lines between each block like above